src/utils/mediaDevicesUtils.js

```diff
@@ -0,0 +1,154 @@
+import store from '@/plugins/vuex';
+import { stopIndicesWithElidedDims } from '@tensorflow/tfjs-core/dist/ops/slice_util';
+
+navigator.mediaDevices.ondevicechange = async function() {
```

Comment:
Move this out of global scope and in an init function, this adds magic and if loading order changes it could break